apps/tasks/lib/pages/login_page.dart

```diff
@@ -67,7 +67,7 @@ class _LoginPageState extends State<LoginPage> {
                   if (email == null || email.isEmpty) {
                     return context.localization!.emailNotValid;
                   }
-                  if (!RegExp(r"^[a-zA-Z0-9.a-zA-Z0-9.!#$%&'*+-/=?^_`{|}~]+@[a-zA-Z0-9]+\.[a-zA-Z]+").hasMatch(email)) {
```

Comment:
I would suggest this package because it would make validating an email easier. The validation of all types of emails could be difficult especially the coverage of all cases can quickly lead to errors with a custom rule expression. https://pub.dev/packages/email_validatort